pkg/migrations/sql/idempotent/telemetry.sql

```diff
@@ -59,4 +59,73 @@ $$
         RETURN TRUE;
     END;
 $$
-LANGUAGE PLPGSQL;
\ No newline at end of file
+LANGUAGE PLPGSQL;
+
+CREATE OR REPLACE FUNCTION SCHEMA_PS_CATALOG.promscale_sql_telemetry() RETURNS VOID AS
```

Comment:
can we reorder this so that we have all metrics telemetry, followed by traces telemetry, followed by general telemetry.